src/components/Mypage/PopupModify.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState, useRef } from "react";
+import { useSetRecoilState } from "recoil";
+import alertAtom from "recoil/alert";
+import ProfileImg from "./ProfileImg";
+import axios from "tools/axios";
+import axiosOri from "axios";
+import convertImg from "tools/convertImg";
+import PropTypes from "prop-types";
+import Modal from "components/common/modal/Modal";
+import DottedLine from "components/common/DottedLine";
+import { theme } from "styles/theme";
+import Button from "components/common/Button";
+
+const ProfImg = (props) => {
+  const style = {
+    margin: "auto",
+    width: "110px",
+    height: "110px",
+    borderRadius: "55%",
+    overflow: "hidden",
+  };
+  return (
+    <div style={style}>
+      {props.profileImgUrl && (
+        <ProfileImg path={props.profileImgUrl} token={props.token} />
+      )}
+    </div>
+  );
+};
+ProfImg.propTypes = {
+  profileImgUrl: PropTypes.string,
+  token: PropTypes.any,
+};
+
+const BtnProfImg = (props) => {
+  const inputImage = useRef(null);
+  const setAlert = useSetRecoilState(alertAtom);
+
+  const handleUploadProfileImage = async () => {
+    try {
+      const image = await convertImg(inputImage.current?.files?.[0]);
+      if (!image) return;
+      axios
+        .post("/users/editProfileImg/getPUrl", { type: image.type })
+        .then(async ({ data }) => {
```

Comment:
@imYourChoi 이런 방식 말씀드린 거였어요!
```suggestion
      const { data } = await axios.post("/users/editProfileImg/getPUrl", { type: image.type });
      // data로 이후 함수 실행
```

Reply:
아하 더 깔끔하네요. 더 수정할 점 있을까요?